resources/js/lang/en.js

```diff
@@ -137,13 +128,6 @@ export default {
                 2: "You've heard reports of large flying creatures in the Copperneck Mountains. Could there really be dragons in the mountains? It's worth investigating, dragons could pose a serious threat to the city.{br}Your goal is the Icecrag, the tallest mountain in the Coppernecks. On your way there, you were ambushed by a group of Inox and Demons. Their co-existence was curious, but your ultimate goal lies further ahead.{br}The climb up the Icecrag was grueling and arduous. You had to fight off the cold, the wolves, and a nest of angry drakes. However you did get to the bottom of this dragon business. Atop the peak was a gigantic winged lizard - that could speak. It spoke with a gravelly voice that surprised you with its eloquence. In fact, it asked for your help. “A group of Savvas subdued me in my sleep and removed one of my flame glands, presumably to study. I cannot let this insult go unpunished and would greatly appreciate you taking it back from their city, killing as many of them as you can in the process.Or, if you wish to be cliché\" the creature said with a sigh, \"you could simply attack me. Slay the dragon, save the princess, and all that. Except I have no princess, and I’m not a dragon — just a drake who has lived a long time and does not wish to end his life today. So if you do attack me, I will fight back, and you will most assuredly die.”"
             }
         },
-        16: {
```

Comment:
This quest was already written about in Quest 12. It made more sense to show that quest for this scenario because it tells you more about the preceding events leading up to the scenario.

Reply:
Scenario 17 can also be accessed from a chest in a different scenario (which in theory you can complete before doing 13), in which case the text from Quest 12 makes no sense. 

Maybe we can make the conditions for deciding which quest to shown more powerful and consider the choice made in scenario 13, but I wouldn't want to remove this one.

Reply:
I didn't know about that treasure, so I agree on your comment and will revert this change.

I don't feel like putting in the effort at this time to improve the conditions for this scenario. It is lacking the 'preceding events' but it does tell you something about the scenario without spoilers. So it's good enough.